Found 20 items
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile10
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile11
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile12
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile13
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile14
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile15
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile16
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile17
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile18
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile19
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile20
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile5
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile6
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile7
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile8
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile9
file 9 has been put into HDFS
file 9 has been read from HDFS for 1 time
diff succeed!
file 9 has been read from HDFS for 2 time
diff succeed!
file 9 has been read from HDFS for 3 time
diff succeed!
file 9 has been read from HDFS for 4 time
diff succeed!
file 9 has been read from HDFS for 5 time
diff succeed!
file 9 has been read from HDFS for 6 time
diff succeed!
file 9 has been read from HDFS for 7 time
diff succeed!
file 9 has been read from HDFS for 8 time
diff succeed!
file 9 has been read from HDFS for 9 time
diff succeed!
file 9 has been read from HDFS for 10 time
diff succeed!
2019-02-28 22:53:20,097 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 11 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 11 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying immediately.
2019-02-28 22:53:20,103 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 11 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 11 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying after sleeping for 720ms.
2019-02-28 22:53:20,832 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 10 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 10 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying after sleeping for 2638ms.
2019-02-28 22:53:23,477 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 8 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 8 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying after sleeping for 5862ms.
2019-02-28 22:53:29,342 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.RetriableException): org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 2 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1441)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.delete(FSNamesystem.java:2986)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.delete(NameNodeRpcServer.java:1095)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.delete(ClientNamenodeProtocolServerSideTranslatorPB.java:692)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
Caused by: org.apache.hadoop.hdfs.server.namenode.SafeModeException: Cannot delete /myfile9. Name node is in safe mode.
The reported blocks 160 has reached the threshold 0.9990 of total blocks 160. The number of live datanodes 3 has reached the minimum number 0. In safe mode extension. Safe mode will be turned off automatically in 2 seconds. NamenodeHostName:node-1-link-0
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.newSafemodeException(FSNamesystem.java:1450)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkNameNodeSafeMode(FSNamesystem.java:1437)
	... 12 more
, while invoking ClientNamenodeProtocolTranslatorPB.delete over node-1-link-0/10.10.1.4:8020. Retrying after sleeping for 4587ms.
Deleted /myfile9
file 9 has been removed from HDFS
Found 17 items
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile10
-rw-r--r--   3 root supergroup  402653184 2019-02-28 22:53 /myfile11._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile12
-rw-r--r--   3 root supergroup  402653184 2019-02-28 22:53 /myfile13._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile14
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile16
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile17
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile18
-rw-r--r--   3 root supergroup  402653184 2019-02-28 22:53 /myfile19._COPYING_
-rw-r--r--   3 root supergroup  134217728 2019-02-28 22:53 /myfile2._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile20
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:53 /myfile6
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:49 /myfile8
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:53 /myfile9
file 9 has been put into HDFS
file 9 has been read from HDFS for 1 time
diff succeed!
file 9 has been read from HDFS for 2 time
diff succeed!
file 9 has been read from HDFS for 3 time
diff succeed!
2019-02-28 22:54:45,725 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From c220g2-011316.wisc.cloudlab.us/128.104.222.136 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1087ms.
2019-02-28 22:54:46,816 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 1555ms.
2019-02-28 22:54:48,374 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From c220g2-011316.wisc.cloudlab.us/128.104.222.136 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 3 failover attempts. Trying to failover after sleeping for 4357ms.
file 9 has been read from HDFS for 4 time
diff succeed!
file 9 has been read from HDFS for 5 time
diff succeed!
file 9 has been read from HDFS for 6 time
diff succeed!
file 9 has been read from HDFS for 7 time
diff succeed!
file 9 has been read from HDFS for 8 time
diff succeed!
file 9 has been read from HDFS for 9 time
diff succeed!
2019-02-28 22:56:51,384 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From c220g2-011316.wisc.cloudlab.us/128.104.222.136 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1293ms.
file 9 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile9
file 9 has been removed from HDFS
2019-02-28 22:57:28,353 INFO retry.RetryInvocationHandler: org.apache.hadoop.ipc.RemoteException(org.apache.hadoop.ipc.StandbyException): Operation category READ is not supported in state standby. Visit https://s.apache.org/sbnn-error
	at org.apache.hadoop.hdfs.server.namenode.ha.StandbyState.checkOperation(StandbyState.java:88)
	at org.apache.hadoop.hdfs.server.namenode.NameNode$NameNodeHAContext.checkOperation(NameNode.java:1952)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.checkOperation(FSNamesystem.java:1423)
	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getFileInfo(FSNamesystem.java:3085)
	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getFileInfo(NameNodeRpcServer.java:1154)
	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getFileInfo(ClientNamenodeProtocolServerSideTranslatorPB.java:966)
	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:523)
	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:991)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:872)
	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:818)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1729)
	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2678)
, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1448ms.
2019-02-28 22:57:29,803 INFO retry.RetryInvocationHandler: java.net.ConnectException: Call From c220g2-011316.wisc.cloudlab.us/128.104.222.136 to node-0-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-0-link-0/10.10.1.1:8020 after 2 failover attempts. Trying to failover after sleeping for 1435ms.
Found 13 items
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:55 /myfile10
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile12
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:55 /myfile14
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile16
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile17
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:55 /myfile18
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile5
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:57 /myfile6
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:54 /myfile8
-rw-r--r--   3 root supergroup 1073741824 2019-02-28 22:57 /myfile9
file 9 has been put into HDFS
file 9 has been read from HDFS for 1 time
diff succeed!
file 9 has been read from HDFS for 2 time
diff succeed!
file 9 has been read from HDFS for 3 time
diff succeed!
file 9 has been read from HDFS for 4 time
diff succeed!
file 9 has been read from HDFS for 5 time
diff succeed!
sub_benchmark 5 signal TERM catched and let quit gracefully
Deleted /myfile9
file 9 has been removed from HDFS
